catkin_virtualenv/src/catkin_virtualenv/venv.py

```diff
@@ -115,6 +115,12 @@ def lock(self, package_name, input_requirements, no_overwrite, extra_pip_args):
         command += ['-o', output_requirements]
 
         check_call(command)
+
+        # Remove user-specific paths
+        contents = open(output_requirements, 'r').read().replace(os.path.dirname(output_requirements), '.')
```

Comment:
No need to open the file twice:
```
with open(output_requirements, 'r+') as f:
    contents = f.read().replace(os.path.dirname(output_requirements) + '/', '')  # `./requirements.in` is a little odd because cwd for this command is the build directory. But I'm not picky, let's just purge the directory entirely.
    f.seek(0)
    f.write(contents)
```